### DISCUSSION TITLE
WARNING: Untransformed solution with objective

### DISCUSSION THREAD

**Discussion by Equal7** [ERGO-Code/HiGHS#1418] [Q&A]
Hi! Can someone help me with my problem. After I run the model calculation, the following text appears in logs
![2023-09-19_16h36_22](https://github.com/ERGO-Code/HiGHS/assets/39560417/30ce8ff2-8cd2-4725-afb0-634b50af73db)
Periodically, but not always following these logs, the optimizer freezes, do not produce any solution or error. 
![image](https://github.com/ERGO-Code/HiGHS/assets/39560417/1640c02b-8614-439d-a4d2-59b1174be90c)
Sometimes, it happens that even a solution is displayed in the logs, and it says that a solution has been found, it is optimal, but the solution function does not return anything, and the application freezes in this state.

**Comment by jajhall** [comment]
Could you let me have the model. Let me know how you are running HiGHS if you are unsure how to do this.

The "freezing" could be the Windows issue that's fixed by setting the option `threads=1`. Again, let me know how you are running HiGHS if you are unsure how to do this.

**Reply by Equal7** [reply]
I'm on Windows and I set the option "threads" = 1, but it doesn't help. I still get the "freezing' effect sometimes after i running HiGHS. In this example i set time_limit = 60, but logs "freeze" after sending last message on 5.2 sec.
![image](https://github.com/ERGO-Code/HiGHS/assets/39560417/26945d9a-51e4-43f8-9bf1-3b8d7b1468e1)
As interface for HIGHS i'm using Pyomo, and sending here my model is quite problematic (too large). But about your point that the problem in my model, probably that really is.

**Reply by jajhall** [reply]
Your model isn't large at all: just a few thousand rows and columns. pyomo should allow you to write out the current model. If I'm to help you, this is what you need to do.

**Reply by Equal7** [reply]
Sorry it's so late. I thought I solved the problem. But now it's happened again. Here is my model. If you would still like to help me, I would really appreciate it
[model.txt](https://github.com/ERGO-Code/HiGHS/files/14307047/model.txt)

**Reply by fwesselm** [reply]
Could you provide the problem in MPS format? Thank you.
